docs/sourcemaps.asciidoc

```diff
@@ -1,38 +1,192 @@
 [[sourcemaps]]
-== Upload source maps (RUM)
+== How to apply source maps to error stack traces when using minified bundles
 
-For a number of reasons, it is common practice to minify JavaScript code, for example to reduce network latency.
-While improving performance, minified code can be hard to debug.
-A source map library helps by mapping the minified files back to the the original source code.
+++++
+<titleabbrev>Create and upload source maps (RUM)</titleabbrev>
+++++
 
-APM Server provides a <<sourcemap-api,Source Map API>> for uploading source maps.
+Minifying JavaScript bundles in production is a common practice;
+it can greatly improve the load time and network latency of your applications.
+The problem with minifying code is that it can be hard to debug.
+Here's an example of an exception stack trace in the APM app when using minified code.
+As you can see, it's not very helpful.
 
-By default, <<config-sourcemapping-enabled>> is set to `true`.
-This means if a source map has previously been uploaded,
-source mapping will automatically be applied all incoming transactions and errors.
+[role="screenshot"]
+image::images/source-map-before.png[APM app without source mapping]
 
+That’s why Elastic APM supports source maps.
+A source map maps minified files back to the original source code, allowing you to maintain the speed advantage of minified code,
```

Comment:
suggestion: "A source map maps minified files..." --> "A source map defines how to convert minified files..."

Reply:
Yeah, "map maps" was not the best choice 😅